packages/encrypt-upload-client/examples/decrypt-test.js

```diff
@@ -61,6 +78,21 @@ async function main() {
     decryptionCapability.with
   )
 
+  const paymentManager = await litClient.getPaymentManager({
+    account: wallet,
+  })
+  const balance = await paymentManager.getBalance({
+    userAddress: wallet.address,
+  })
+  console.log('Current Balance: ', balance.totalBalance)
+  // const depositReceipt = await paymentManager.deposit({
```

Comment:
are these comments still here intentionally? would be great to have a brief comment explaining why if so